### ISSUE TITLE
form field added by ConversationalForm.addTags not included in form submit

### ISSUE THREAD

**Issue by tstkenny** [space10-community/conversational-form#236]
Hi there.  I setup flowStepCallback to add a question in runtime and submit the form by  
$('#form').serialize().  However the question added does not being included in the form submit.  Here's how I added the question and it works in the conversation properly:

window.ConversationalForm.addTags([
                            {
                                tag: "select",
                                name: "Booking[idproduct]",
                                "cf-questions":
                                        "Please select a product",
                                "cf-input-placeholder": "",
                                multiple: false,
                                children: children,
                            }
                        ]);

I checked the DOM and there is no new element of <fieldset></fieldset> added as well.

Thank you very much and keep it up this fantastic project!

**Comment by jenssogaard** [comment]
@tstkenny you are right, it's not added to the DOM. You should have a look at [this part of the docs](https://space10-community.github.io/conversational-form/docs/0.9.80/functionality/submit-callback/).

`var formDataSerialized = myConvFormInstance.getFormData(true);`
